bridge/contracts/Snapshots.sol

```diff
@@ -60,13 +62,19 @@ contract Snapshots is Initializable, SnapshotsStorage, ISnapshots {
             IValidatorPool(_validatorPoolAddress()).isConsensusRunning(),
             string(abi.encodePacked(SnapshotsErrorCodes.SNAPSHOT_CONSENSUS_RUNNING))
         );
-
+        //get the last snapshot
+        Snapshot memory lastSnapshot = getLatestSnapshot();
         require(
-            block.number >= _snapshots[_epoch].committedAt + _minimumIntervalBetweenSnapshots,
+            block.number >= lastSnapshot.committedAt + _minimumIntervalBetweenSnapshots,
             string(abi.encodePacked(SnapshotsErrorCodes.SNAPSHOT_MIN_BLOCKS_INTERVAL_NOT_PASSED))
         );
-
-        uint32 epoch = _epoch + 1;
+        Epoch storage epochReg = _epochReg();
```

Comment:
What is epochReg?

Reply:
sorry about that, I forgot to take out that add but epochReg is the function that returns the struct that holds the epoch. It is implemented in snapshotstorage.sol